kibana-reports/public/components/report_definitions/report_settings/report_settings.tsx

```diff
@@ -91,6 +101,17 @@ function TimeRangeSelect() {
     );
     setIsLoading(true);
     startLoading();
+
+    // parseTimeRange(start, end, reportDefinitionRequest);
```

Comment:
If this is unused can we remove this comment

Reply:
sure